src/main/java/us/tastybento/bskyblock/api/configuration/ConfigEntry.java

```diff
@@ -16,7 +16,7 @@
 @Target(ElementType.FIELD)
 public @interface ConfigEntry {
 
-    String path();
+    String path() default "";
```

Comment:
There should not be a default here. A path MUST be specified for each ConfigEntry, having a default value would cause a lot of issues.

Reply:
Yes. In that case, I need to split off Adapter from ConfigEntry because Adapter is used for non-YAML storage too. I’ll make it it’s own Notation.